api/src/main/java/org/pmiops/workbench/api/BugReportController.java

```diff
@@ -36,7 +36,7 @@
       msg.setFrom(new InternetAddress("all-of-us-workbench-eng@googlegroups.com"));
       msg.addRecipient(Message.RecipientType.TO, new InternetAddress("all-of-us-workbench-eng@googlegroups.com", "AofU Workbench Engineers"));
       msg.setSubject("[AofU Bug Report]: " + bugReport.getShortDescription());
-      msg.setText(bugReport.getReproSteps());
+      msg.setText(bugReport.getReproSteps() + "\n\n\nTo contact the reporter of this bug, email:\n" + bugReport.getContactEmail());
```

Comment:
Would setting the reply-to address as the reporter's address work?

Reply:
Yup, will make that change.